src/canmatrix/cli/compare.py

```diff
@@ -51,23 +53,29 @@ def cli_compare(matrix1, matrix2, verbosity, silent, check_comments, check_attri
         matrixX can be any of *.dbc|*.dbf|*.kcd|*.arxml|*.xls(x)|*.sym
     """
 
-    import canmatrix.log
-    root_logger = canmatrix.log.setup_logger()
+    root_logger = canmatrix_log.setup_logger()
 
     if silent:
         # Only print ERROR messages (ignore import warnings)
         verbosity = -1
-    canmatrix.log.set_log_level(root_logger, verbosity)
+    elif not verbosity:
+        # Info Message
+        verbosity = 1
+    else:
+        # Debug Message
+        verbosity = 2
+
+    canmatrix_log.set_log_level(root_logger, verbosity)
 
     # import only after setting log level, to also disable warning messages in silent mode.
-    import canmatrix.formats  # due this import we need the import alias for log module
+    import canmatrix.formats as canmatrix_format # due this import we need the import alias for log module
```

Comment:
what is the benefit of importing with an alias-name?

Reply:
Yep, Actually not usally here. During the test, I just found the import does not work, so I tried to use as alias
Change can be reverted.